src/DebugEngineHost/packages.config

```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.VisualStudio.ComponentModelHost" version="15.0.0" targetFramework="net45" />
```

Comment:
I don't think you need this file. I don't think we build this assembly, or we should not be, on Desktop.* builds.